### ISSUE TITLE
belongsToMany won't create correct pivot table on certain case

### ISSUE THREAD

**Issue by kiaking** [vuex-orm/vuex-orm#137]
The pivot table for the `belongsToMany` relation won't get created if the name of the key that defines the relation is different from the entity name of the related model.

```js
class User extends Model {
  static fields () {
    return {
      id: this.attr(''),
      members: this.belongsTo(Group, GroupUser, 'user_id', 'group_id') // <- Doesn't work!
    }
  }
}

class Group extends Model {
  static fields () {
    return {
      id: this.attr('')
    }
  }
}
```

**Comment by kiaking** [comment]
This issue has been fixed by #139 and will be included in the next release.

**Comment by kiaking** [comment]
This issue is now resolved with Vuex ORM 0.24.2!
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.24.2
